makefile

```diff
@@ -14,8 +14,9 @@ EXT_DIR            := ./.ext
 EXT_BIN_DIR        := ${EXT_DIR}/bin
 EXT_TMP_DIR        := ${EXT_DIR}/tmp
 
+GO_VERSION         := 1.23
```

Comment:
Please add the go version check to the makefile, but add it to deps as there is no build